Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -387,8 +386,28 @@ public T CreateInstance<T>()
         /// <returns>System.Object.</returns>
         protected object CreateInstanceSafe(Type type)
         {
+            if (_creatingInstances == null)
+            {
+                _creatingInstances = new List<Type>();
+            }
+
+            if (_creatingInstances.IndexOf(type) != -1)
+            {
+                Logger.LogError("DI Loop detected.");
+                Logger.LogError("Attempted creation of {Type}", type.FullName);
```

Comment:
Logging twice is not good. It's annoying if you're using a log aggregator as it can count as two separate events.

Reply:
Corrected